homeassistant/components/fyta/entity.py

```diff
@@ -20,7 +21,7 @@ def __init__(
         self,
         coordinator: FytaCoordinator,
         entry: FytaConfigEntry,
-        description: SensorEntityDescription,
+        description: SensorEntityDescription | BinarySensorEntityDescription,
```

Comment:
(I originally skipped this, but now that I still have one comment, let's just mention this as well)

This can just be typed `EntityDescription`